dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -676,6 +679,14 @@ private Token createDockstoreToken(long userID, String githubLogin) {
         return dockstoreToken;
     }
 
+    public boolean restrictUser(String username) {
```

Comment:
Nit: this name is confusing, sounds like an action that it will restrict the user.

Reply:
Agreed, this should sound like a predicate that calculates whether the user should be locked down.

Reply:
changed